src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -30526,6 +30625,19 @@ unit:MilliOHM-PER-M
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "milliohm per metre" .
 
+unit:MilliOSM-PER-KiloGM
```

Comment:
Theoretically, the unscaled unit would be OSM per kg, so the conversionMultiplier cannot be 1 here - it must be 0.001